terragrunt/aws/cloudfront/cloudfront.tf

```
@@ -0,0 +1,95 @@
+resource "aws_cloudfront_distribution" "simplify_privacy_app_cf_distribution" {
+  enabled     = true
+  aliases     = [var.domain]
+  price_class = "PriceClass_100"
+  web_acl_id  = aws_wafv2_web_acl.api_waf.arn
+
+  origin {
+    domain_name = split("/", var.generated_statement_lambda_function_url)[2]
+    origin_id   = var.generated_statement_lambda_function_url_name
+
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_read_timeout    = 60
+      origin_protocol_policy = "https-only"
+      origin_ssl_protocols   = ["TLSv1.2"]
+    }
+  }
+  default_cache_behavior {
+    allowed_methods = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+    cached_methods  = ["GET", "HEAD"]
+
+    forwarded_values {
+      query_string = true
+      headers      = ["Authorization"]
```

You can remove this since this app doesn't have an `Authorization` header that we need to allow through to the origin.  This would be for something like an API that needed an `Authorization: some-api-key` header value to authenticate users.
```suggestion
      headers      = []
```